Celeste.Mod.mm/Patches/CrystalStaticSpinner.cs

```diff
@@ -47,5 +165,55 @@ public CoreModeListener(CrystalStaticSpinner parent)
             }
         }
 
+        // idk how to fix "CrystalStaticSpinner.Border is innaccesible due to it's protection level", so I'll just do this.
```

Comment:
From what I can see, this would not be an issue at all if the texture lookup got changed either by modifying the dictionary, or by using IL hooks.